src/style/MystheaUniverse/ToolBar.qml

```diff
@@ -9,15 +9,16 @@ T.ToolBar {
     id: control
 
     property alias separatorColor: _separator.color
+    property alias backgroundColor: _rectangle.color
```

Comment:
The `CardsFiltersHeader` too is a ToolBar so you have to add this color in that component too. 
You can find this component under `MystheaUniverse/Components`.